exome_alignment.cwl

```diff
@@ -17,7 +17,7 @@ inputs:
     known_indels:
         type: File
         secondaryFiles: [.tbi]
-    dbsnp:
+    dbsnp_vcf:
```

Comment:
If we're renaming this one, should we also be changing `known_indels` and `mills`? (`omni_vcf` already has "vcf" in the name.)  I suppose a general question: Is it helpful including filetypes in input names?  We could rename `reference` to `reference_fasta` while we're at it!  (I think I'd argue against doing that... but what does that mean for the others?)

Reply:
I agree that we should look for consistency here! I did some quick greps and must have missed some.  

My general rule has been - if I'm putting together a yaml file and it's not clear what file is needed, then more specificity is needed.  There are a variety of different dbSNP formats, but a reference is almost certainly a fasta (that said, I am generally in favor of being more descriptive, not less, so reference_fasta would be fine!).  

Can we go ahead and punt additional renaming to a separate PR, since this one is bloated enough?

Reply:
Yeah--we can certainly worry about renaming other inputs later!  I'll gladly +1 after my other comment is dealt with (which should ensure Travis CI is satisfied) 😄